pkg/plugin/work-in-progress/plugin/event_handler_unit_test.go

```diff
@@ -33,6 +34,7 @@ var _ = Describe("Work-in-progress Plugin features", func() {
 		DescribeTable("should not recognize PR as work-in-progress if title doesn't start with WIP",
 			func(title string) {
 				Expect(handler.IsWorkInProgress(&title)).To(BeFalse())
+				Expect(handler.BotName).To(Equal("alien-ike"))
```

Comment:
I would think about testing it on `GitHub` level checking if status has proper "context" field instead. WDYT @MatousJobanek?

Here we simply test if constructor works, not sure if that's the optimal route.

Reply:
I do agree with you @bartoszmajsak - this should have been verified on the level of the post request sent by `testStatusService` - @hemanik  take a look here: https://github.com/arquillian/ike-prow-plugins/blob/master/pkg/plugin/test-keeper/plugin/event_handler_gh_test.go#L26